src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -35,7 +35,7 @@ import { pathExists } from '../../helpers/filesystem'
 // The UTF-8 characters "h", "t", "t", and "p".
 const HTTP_IN_HEX = 0x68747470
 
-const USE_OVERFLOW_MENU_WIDTH_THRESHOLD = 600
+const USE_OVERFLOW_MENU_WIDTH_THRESHOLD = 634
```

Comment:
im a bit confused what changed here. Im seeing the overflow spawn around 663

![image](https://github.com/user-attachments/assets/45ae3a7c-20d2-4b32-b4a2-afa633d6b0fd)

Reply:
It's not referring to the full viewport width:
https://github.com/FreeTubeApp/FreeTube/blob/0e6ce22ed33e0dbcc6804a5f98fae0cf8d6d9e28/src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js#L982-L984